A/ASL/build_tarballs.jl

```diff
@@ -0,0 +1,92 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "ASLBuilder"
+version = v"0.1.0"
+
+# Collection of sources required to build ThinASLBuilder
+sources = [
+    "http://netlib.org/ampl/solvers.tgz" =>
+    "775b92cadaf95af73fdeec3effba6b9c6ffdc518e5f628b575140fb170885903",
+    "./bundled"
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd solvers/
+mkdir $prefix/lib
+
+so="so"
```

Comment:
You can use the environment variable `${dlext}` which holds the right extension of the dynamic library for each platform, instead of conditionally setting everywhere